docs/features/software-templates/installation.md

```diff
@@ -180,8 +189,18 @@ Apps integration further down the line.
 > **Right now it is only possible to scaffold repositories inside GitHub
 > organizations, and not under personal accounts.**
 
-The GitHub access token is passed along using the `GITHUB_ACCESS_TOKEN`
-environment variable.
+The Github access token is retrieved from environment variables by the config.
+The config file needs to specify what environment variable the token is
+retrieved from. Your config should have the following objects.
+
+```yaml
+scaffolder:
```

Comment:
@Rugvip is there a global config with this key somewhere?

Reply:
You mean if the top-level scaffolder key is used anywhere else? Not sure, but seems like a sensible choice